src/fragments/lib/ssr/js/getting-started.mdx

```diff
@@ -75,6 +75,38 @@ Server-side functions that don't have a req object (e.g. Next.js' getStaticProps
 
 </Callout>
 
+## Next.js App Router Usage
+Amplify JS can be used with Next.js' App Router with a few important changes to how you configure and use the library, namely:
```

Comment:
```suggestion
Amplify JS can be used with Next.js' App Router with a few important changes on how you configure and use the library, namely:
```

Reply:
In this case I think `to` reads a bit better (the changes are to the process by which you configure & use the library)